pwiz_tools/Skyline/SkylineGraphs.cs

```diff
@@ -5539,6 +5539,11 @@ public LiveResultsGrid CreateResultsGrid()
             return _resultsGridForm;
         }
 
+        public LiveResultsGrid GetLiveResultsGrid()
```

Comment:
Probably best to do this as a property as we have done with other forms like this. I can't think of another that uses a function.

Reply:
Makes sense - good call.